volatility3/framework/plugins/windows/pe_symbols.py

```diff
@@ -661,15 +662,25 @@ def _get_symbol_value(
             # address or name
             if symbol_key in wanted_symbols:
                 # walk each wanted address or name
-                for value_index, wanted_value in enumerate(wanted_symbols[symbol_key]):
-                    symbol_value = symbol_getter(wanted_value)
 
+                # build dict in this function for debugging and tracking
+                all_wanted = []
+                for wanted_value in wanted_symbols[symbol_key]:
+                    all_wanted.append(wanted_value)
+
+                for value_index, wanted_value in enumerate(all_wanted):
+                    symbol_value = symbol_getter(wanted_value)
                     if symbol_value:
                         # yield out deleteion key, deletion index, symbol name, symbol address
                         if symbol_key == wanted_names_identifier:
-                            yield symbol_key, value_index, wanted_value, symbol_value  # type: ignore
+                            yield symbol_key, wanted_value, symbol_value  # type: ignore
```

Comment:
Don't suppose we can do anything about those type: ignore things can we?  Given we're modifying the code, I'd like to just check if they're still need or not (now things are typed a little better and  the checkers are more advnaced)...